src/components/Shared/DatePicker.vue

```diff
@@ -221,6 +218,23 @@ export default {
 
 			return timeFormat.indexOf('a') !== -1
 		},
+		formatter() {
+			// Wrap moment constructor as this.locale is reactive/async
```

Comment:
```suggestion
			// Wrap moment constructor as this.locale is reactive
```

you got me confused with async-await. the locale may change but it is accessed synchronously